### DISCUSSION TITLE
All services goes unreachable when starting a torrent download on Casa

### DISCUSSION THREAD

**Discussion by petersyntek** [IceWhaleTech/CasaOS#1875] [Q&A]
Hi Everyone,

I have Debian 11 on my dad's homeserver, with OpenMediaVault and CasaOS. Jellyfin, Plex and Cloudflared running in Casa. Today I tried to setup a torrent client, but I run into an issue.

Everything working fine util a download stats in a torrent client, from that point all services and web interfaces are unavailable, cannot ping the server or SSH into it, till the download finishes. I tried with qBit, Deluge and also Transmission, tried different network, cpu, memory setting in the container, but nothing worked.

Did anyone there meet this issue?

**Comment by TheCorruptCat** [comment]
I have the same issue but it just happens randomly for me

**Comment by TheCorruptCat** [comment]
https://github.com/IceWhaleTech/CasaOS/issues/1876

**Reply by petersyntek** [reply]
Hi! Thank for mentioning your topic, but I only have the issue when a torrent download is running. I haven't got to much time for debug since I posted, but I've tried to apply bandwidth limit on the download, but also not helped with the issue. Just was unavailable for a longer time due to the slower download speed.

**Comment by petersyntek** [comment]
It turned out, in my case it's seems not an issue with CasaOS. I've installed Transmission directly onto the Debian 11, that runs OMV and CasaOS and from my dad's PC issue was the same. When a download started all web interfaces and services seems unreachable. But this time a brought over may laptop which runs the latest Fedora, and guess what? From that machine was everything reachable. I was able to access web interfaces, shares, also Transmission via rpc with the remote GUI. Now what? :D I'll need to bring over an other Windows machine and test with that.
